web/lib/HabitatCommunityPreviewCreator.js

```diff
@@ -68,23 +114,43 @@ export default class HabitatCommunityPreviewCreator extends HTMLElement {
     this._ctx.canvas.height = h;
 
     wrapListener(this.shadowRoot.querySelector('#create'), this.create.bind(this));
+    wrapListener(this.shadowRoot.querySelector('#boxleg'), () => {
+      //flip "create [community]" button to active / inverted color
+      this.parentElement.parentElement.parentElement.querySelector('#create-community').classList.toggle('active');
```

Comment:
A special CSS variable should be used to change visibility. Otherwise this throws If the layout of the dom changes or this element will be used somewhere else.